server/actions_apis_test.go

```diff
@@ -16,6 +16,34 @@ import (
 	"google.golang.org/protobuf/types/known/fieldmaskpb"
 )
 
+var (
+	// Basic API view does not include annotations.
+	basicApi = &rpc.Api{
+		Name:               "projects/my-project/apis/my-api",
+		DisplayName:        "My Api",
+		Description:        "Api for my versions",
+		Availability:       "GENERAL",
+		RecommendedVersion: "v1",
+		Labels: map[string]string{
+			"label-key": "label-value",
+		},
+	}
+	// Full API view includes annotations.
+	fullApi = &rpc.Api{
+		Name:               "projects/my-project/apis/my-api",
+		DisplayName:        "My Api",
+		Description:        "Api for my versions",
+		Availability:       "GENERAL",
+		RecommendedVersion: "v1",
+		Labels: map[string]string{
+			"label-key": "label-value",
+		},
+		Annotations: map[string]string{
+			"annotation-key": "annotation-value",
+		},
+	}
+)
+
```

Comment:
I think this structure is great, and just wonder - do you see an advantage to building both views explicitly instead of deriving one from a copy of the other?

Reply:
I think simplicity is the primary benefit. Ideally the data for each test case shouldn't require any code execution to produce because it makes it more difficult to understand the test.

It would be better if we could define these in-line so readers don't have to jump around the file to understand the test, but fully populated structs across several test cases would be difficult to maintain if fields are added. This was the best compromise I could come up with.